src/deprecated.jl

```diff
@@ -2,3 +2,24 @@ typealias EmptyStreamSource EmptyStream
 
 Base.@deprecate EmptyStreamSource EmptyStream
 Base.@deprecate seekforward(stream::BufferedInputStream, n::Integer) skip(stream, n)
+
+# emptystream.jl 
+
+if VERSION < v"0.6-" 
+    function Base.takebuf_array(stream::BufferedOutputStream{EmptyStream})
+        chunk = stream.buffer[1:stream.position-1]
+        stream.position = 1
+        return chunk
+    end
+end
+
+if v"0.5-" < VERSION < v"0.6-"
```

Comment:
I guess that as simple a change as `v"0.5-"` -> `v"0.5"` should be sufficient to fix this?